firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -223,8 +223,7 @@ def tearDown(self):
 
     @skip_under_xvfb
     def test_elements(self):
-        self.locationbar.identity_box.click()
-        Wait(self.marionette).until(lambda _: self.identity_popup.is_open)
+        self.open_identity_popup()
```

Comment:
Please keep in mind that `self` in this context is the test class but not the instance of the locationbar. What you have to use here is `self.locationbar.open_identity_popup()`. Please ensure to really run all tests locally and check travis results before asking for review/feedback. If there are questions please let me know via a need-info? request in Bugzilla.